spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -5,9 +5,29 @@
     request.accept = 'application/json'
   end
 
+  describe '#show' do
+    before do
+      @user = create :user_with_team, :chris
+    end
+
+    it 'returns user data' do
+      get :show, id: @user.id
+
+      expect(response).to be_success
+      expect(json['id']).to eq(@user.id)
+      expect(json['username']).to eq(@user.username)
+      expect(json['country']).to eq(@user.country)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.